File: internal/content/testdata/bench/rust/nc-coin-change-ii.rs
fn change(amount: i64, coins: Vec<i64>) -> i64 {
    let amt = amount as usize;
    let mut dp = vec![0i64; amt + 1];
    dp[0] = 1;
    for &coin in coins.iter() {
        let c = coin as usize;
        for x in c..=amt {
            dp[x] += dp[x - c];
        }
    }
    dp[amt]
}

File: internal/content/testdata/bench/javascript/nc-coin-change-ii.js
function change(amount, coins) {
    const dp = new Array(amount + 1).fill(0);
    dp[0] = 1;
    for (const coin of coins) {
        for (let x = coin; x <= amount; x++) {
            dp[x] += dp[x - coin];
        }
    }
    return dp[amount];
}
